fizzbuzz.js

```diff
@@ -0,0 +1,17 @@
+for( var i = 1; i <= 100; i++ ) {
+  var divisibleByThree = i % 3 === 0;
+  var divisibleByFive = i % 5 === 0;
```

Comment:
you should only really have assignment or calculations going on in a statement, not both.